cmd/summary.go

```diff
@@ -36,12 +37,33 @@ func init() {
 }
 
 var summaryCmd = &cobra.Command{
-	Use:   "summary",
-	Short: "Genarate a summary of the vpa recommendations in a namespace.",
-	Long:  `Gather all the vpa data in a namespace and generaate a summary of the recommendations.`,
+	Use:   "summary [namespace]",
+	Short: "Generate a summary of the vpa recommendations in a namespace (default: all namespaces).",
+	Long: `Gather all the vpa data in a namespace and generate a summary of the recommendations.
+By default the summary will be about all VPAs in all namespaces.`,
+	Args: cobra.ArbitraryArgs,
 	Run: func(cmd *cobra.Command, args []string) {
+		var opts []summary.Option
+
+		// limit to a single namespace
+		if len(args) == 1 {
+			opts = append(opts, summary.ForNamespace(args[0]))
+		} else if len(args) > 1 {
+			// undefined argument behaviour
```

Comment:
I don't think this comment is necessary. The klog.Fatal is information enough here.